server/cli/launch.py

```diff
@@ -164,14 +168,27 @@ def launch(
     else:
         port = find_available_port(host)
 
+    if about:
+        def url_check(url):
+            try:
+                result = urlparse(url)
+                if all([result.scheme, result.netloc]):
+                    return True
+                else:
+                    return False
+            except ValueError:
+                return False
+        if not url_check(about):
+            raise click.ClickException("Incorrect URL specified for --about. (Example format: http://example.com)")
+
     # Setup app
     cellxgene_url = f"http://{host}:{port}"
 
     # Import Flask app
     server = Server()
 
     server.create_app()
-    server.app.config.update(SCRIPTS=scripts)
+    server.app.config.update(SCRIPTS=scripts, ABOUT_DATASET=about)
```

Comment:
Why is about URL passed in both the cellxgene `config` and in the flask template variables?    I believe you only use the former, so you can probably delete the flask template (this change).